ecl/hql/hqlmanifest.cpp

```diff
@@ -29,12 +30,27 @@ class ResourceManifest : public CInterface
     {
         try
         {
-            manifest.setown(createPTreeFromXMLFile(filename));
+            fileContents.loadFile(filename, false);
+            const char *xml = fileContents.str();
+            StringBuffer body;
+            // Check for signature
+            if (startsWith(fileContents, "-----BEGIN PGP SIGNED MESSAGE-----"))
+            {
+                // OwnedHqlExpr sig = checkSignature(fileContents.length(), fileContents.str());
```

Comment:
Shouldn't it be checking this?

Reply:
No - you don't want to check when creating an archive. That would require all users to install the keys required to validate signatures. In "local repository" mode the keys are only on the server.

Reply:
Should probably remove the commented-out code though (and replace with a comment about why I don't check)